public/settings.json

```diff
@@ -2,7 +2,13 @@
   "environment": "Local",
   "google": {
     "clientId" : "147813579094-8qbv6canv6167qlga4k9788dg0qmgokn.apps.googleusercontent.com",
-    "cookiePolicy": "single_host_origin"
+    "cookiePolicy": "single_host_origin",
+    "map": "AIzaSyBZ5t__ofNKveXOiPmgwX809LR8o7P8BSo",
```

Comment:
Let's create a single complex `map` field here, as it makes more sense to have all map related information within a single object, like this:
![image](https://user-images.githubusercontent.com/23569938/129867309-fd1030e0-fbb5-42cd-81a2-95395547cb31.png)

Reply:
Done!